pupil-api/package.json

```diff
@@ -18,24 +16,16 @@
   },
   "dependencies": {
     "applicationinsights": "^1.0.2",
-    "bluebird": "^3.5.1",
     "body-parser": "^1.18.2",
     "cors": "^2.8.4",
     "dotenv": "^4.0.0",
     "express": "^4.16.2",
     "express-winston": "^2.5.1",
     "helmet": "^3.12.0",
-    "jsonwebtoken": "^8.2.1",
     "moment": "^2.22.1",
     "morgan": "^1.9.0",
-    "mssql": "^4.1.0",
-    "postgrator": "^3.6.0",
-    "ramda": "^0.25.0",
-    "random-number-csprng": "^1.0.2",
-    "tedious": "^2.3.1",
-    "tedious-connection-pool": "^1.0.5",
     "to-bool": "0.0.1",
-    "uuid": "^3.1.0",
+    "uuid": "^3.3.2",
```

Comment:
you will also need `azure-storage` package

Reply:
That won't be needed until the PR that implements the lookup though, which is planned as a subsequent PR.  This PR is just a tidy up.

Reply:
good point